server/v2/store/config.go

```diff
@@ -0,0 +1,21 @@
+package store
+
+import (
+	storev2 "cosmossdk.io/store/v2"
+)
+
+func DefaultConfig() *Config {
+	return &Config{
```

Comment:
i think we are missing things from here. do we plan on adding it later?

Reply:
do u mean other fields of `BaseConfig`, currently just add config for prune.